analyses/cell-type-consensus/Dockerfile

```diff
@@ -1,10 +1,40 @@
-# A template docker file for creating a new analysis
-FROM ubuntu:22.04
+# Dockerfile for cell-type-consensus module
+FROM bioconductor/r-ver:3.20
```

Comment:
noting we probably want 3.19 which is our preferred OpenScPCA-wide version.

Reply:
Based on this [comment](https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/936#discussion_r1889030481) we are okay with 3.20 here.